src/modules/social/profiles/profiles.route.ts

```diff
@@ -0,0 +1,67 @@
+import { FastifyInstance } from "fastify"
+
+import { $ref } from "./profiles.schema"
+import {
+  getProfilesHandler,
+  getProfileHandler,
+  updateProfileHandler
+} from "./profiles.controller"
+
+async function profilesRoutes(server: FastifyInstance) {
+  server.get(
+    "/",
+    {
+      preHandler: [server.authenticate],
+      schema: {
+        tags: ["profiles"],
+        security: [{ bearerAuth: [] }],
+        response: {
+          200: {
+            type: "array",
+            items: $ref("displayProfileSchema")
+          }
+        }
+      }
+    },
+    getProfilesHandler
+  )
+
+  server.put(
+    "/:id",
+    {
+      preHandler: [server.authenticate],
+      schema: {
+        body: $ref("createProfileSchema"),
```

Comment:
We are missing the required id param in the schema here.

```suggestion
      schema: {
        params: {
          type: "object",
          properties: {
            id: { type: "integer" }
          }
        },
        body: $ref("createProfileSchema"),
```